### ISSUE TITLE
Dispatching an event when files are changed

### ISSUE THREAD

**Issue by timonson** [cmorten/refresh#1]
What do you think about the idea of dispatching a global event, maybe in line 91, so that we could add additional listeners for development tools?
Thank you!

**Comment by cmorten** [comment]
Yeah sure, sounds reasonable 🙂 if have a particular idea feel free to raise a PR!
